### PR TITLE
Add remote sponsor

### PR DESCRIPTION
**Pull request by pablocostass**
I also took the chance to change how we quantify prizes, because now that we have stuff with numbers on their names it becomes a bit hard/weird to read for me. So I propose that "5 PlayStation 2" should be "5x PlayStation 2"

### CODE REVIEW

**Comment by paulo-ferraz-oliveira** [comment]
> I also took the chance to change how we quantify prizes, because now that we have stuff with numbers on their names it becomes a bit hard/weird to read for me. So I propose that "5 PlayStation 2" should be "5x PlayStation 2"

Somebody's handing out 5 PS2? Isn't that pre-you-were-born? 🤣

**Comment by pablocostass** [comment]
> > I also took the chance to change how we quantify prizes, because now that we have stuff with numbers on their names it becomes a bit hard/weird to read for me. So I propose that "5 PlayStation 2" should be "5x PlayStation 2"
> 
> Somebody's handing out 5 PS2? Isn't that pre-you-were-born?

Excuse you, but I grew up playing a lot of Ratchet & Clank on a PS2 :joy:

File: _config.yml [hunk 1]
```diff
@@ -127,22 +131,26 @@ prizes:
...
     sponsor: Groxio
     sponsor_url: https://grox.io
     image: prizes_2021/passport.png
-  - name: 1 reMarkable 2
+  - name: 1x reMarkable 2
     sponsor: Nomasystems
     sponsor_url: http://www.nomasystems.es/
     image: prizes_2021/remarkable2.png
+  - name: 4x Oculus Quest 2
+    sponsor: Remote.com
```
Comment (paulo-ferraz-oliveira) [inline _config.yml:151]:
```suggestion
    sponsor: Remote
```

**Review by paulo-ferraz-oliveira — APPROVED** [review]

**Comment by paulo-ferraz-oliveira** [comment]
Awesome stuff.

**Review by starbelly — APPROVED** [review]
Nice 👍

**Pull request merged**